pkg/data/crd.go

```diff
@@ -31,6 +32,19 @@ func createCRDs(ctx context.Context, restConfig *rest.Config) error {
 			crd.NonNamespacedFromGV(rancherv3.SchemeGroupVersion, "Token", rancherv3.Token{}),
 			crd.NonNamespacedFromGV(rancherv3.SchemeGroupVersion, "NodeDriver", rancherv3.NodeDriver{}),
 			crd.NonNamespacedFromGV(upgradev1.SchemeGroupVersion, "Plan", upgradev1.Plan{}),
+			crd.NonNamespacedFromGV(devices.SchemeGroupVersion, "PCIDevice", devices.PCIDevice{}).
+				WithColumn("Address", ".status.address").
+				WithColumn("Vendor Id", ".status.vendorId").
+				WithColumn("Device Id", ".status.deviceId").
+				WithColumn("Node Name", ".status.nodeName").
+				WithColumn("Description", ".status.description").
+				WithColumn("Kernel Driver In Use", ".status.kernelDriverInUse"),
+			crd.NonNamespacedFromGV(devices.SchemeGroupVersion, "PCIDeviceClaim", devices.PCIDeviceClaim{}).
+				WithColumn("Address", ".spec.address").
+				WithColumn("Node Name", ".spec.nodeName").
+				WithColumn("User Name", ".spec.userName").
+				WithColumn("Kernel Driver Το Unbind", ".status.kernelDriverToUnbind").
+				WithColumn("Passthrough Enabled", ".status.passthroughEnabled"),
```

Comment:
we can leave the column empty if the pci-device controller will overwrite the CRD eventually so we don't need to manage the update in the future.